spec/api-browser-window-spec.js

```diff
@@ -1959,7 +1962,7 @@ describe('BrowserWindow module', function () {
         assert.equal(w.isResizable(), true)
       })
 
-      it('works for a frameless window', () => {
+      xit('works for a frameless window', () => {
```

Comment:
This should be noted somewhere that it was disabled and needs to be re-enabled before `1.8.x` can go stable